narwhals/_duckdb/expr.py

```diff
@@ -99,6 +99,18 @@ def __narwhals_namespace__(self) -> DuckDBNamespace:  # pragma: no cover
 
         return DuckDBNamespace(version=self._version)
 
+    def __floordiv__(self, other: Any) -> Self:
+        def func(expr: Expression, other: Expression) -> Expression:
+            return when(other != lit(0), expr // other).otherwise(lit(None))
+
+        return self._with_binary(func, other=other)
```

Comment:
I think this may need you to define `window_func`, like we do for `Expr.sum` - otherwise would `nw.col('a') / nw.col('a').sum()` work?

(ok to keep as a follow-up, but if it doesn't work, we should track it)